requirements.txt

```diff
@@ -3,3 +3,6 @@ netCDF4
 pyproj
 uptide
 pytz
+networkx
+pyyaml
+sympy
```

Comment:
At the minute the implementation converts yml->dict->sympy->UFL. Maybe it is possible to do yml->dict->UFL? If so, it's probably best to leave for future work.